### DISCUSSION TITLE
oh oops I only have 300 build minutes 😂

### DISCUSSION THREAD

**Discussion by huynhtrankhanh** [facebook/docusaurus#6013] [Q&A] [answered]
hmm so I'm on the Netlify free plan and I only have 300 build minutes each month. each build takes about 30s to 1min apparently. I think I'm afraid of running out of build minutes 😂

hmm so does Docusaurus have some sort of caching mechanism that I could use with Netlify, similar to what Next.js does? by default Netlify only caches dependencies, not build artifacts 😂

**Comment by Josh-Cena** [comment, marked as answer]
> each build takes about 30s to 1min apparently.

That's super fast—the speed you would expect with caching. (And I suspect Netlify automatically caches the artifacts, since they do live in `node_modules`.) FYI my builds on a CI service where I didn't set up caching at all takes about 10min per locale (and it's medium sized, ~100 pages).

It's pretty... remarkable that you made 300+ builds in one month. Is it because of deploy preview + a bunch of dependency upgrade / incremental work?

**Reply by slorber** [reply]
FYI Vercel has 6000m build mins for hobby projects
